### PR TITLE
check for -head.fif in plot_alignment

### LINKED ISSUE #4783: BUG plot_alignment doesn't use `-head.fif` even when exists
Using the default `surfaces` parameter (`'head'`), `plot_alignment` 

```python
mne.viz.plot_alignment(info=fake_info, trans=trans, subject=subject, dig=True,
                       meg=False, src=src, subjects_dir=subjects_dir,
                       surfaces='head')
```

raises an

```python
IOError: No head surface found for subject fsaverage after trying:
/Users/cjb/tmp/subs_dir/fsaverage/bem/outer_skin.surf
/Users/cjb/tmp/subs_dir/fsaverage/bem/flash/outer_skin.surf
```

even though `-head.fif` exists in `bem`. I think there's a simple PR for this...

Context: ran into this when replying to a mail on the list, don't often deal with subjects for which `outer_skin` doesn't exist at this stage of analysis.

### PR DESCRIPTION
**Pull request by cjayb**
Code could be simplified further, bit verbose now, but perhaps best left so (assuming bots agree!)

Closes #4783

### CODE REVIEW

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -683,11 +683,13 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
             if head_surf is None:
                 subject_dir = op.join(
                     get_subjects_dir(subjects_dir, raise_error=True), subject)
-                if s in ('head-dense', 'seghead'):
+                if 'head' in s:  # 'head-dense', 'seghead', 'head'
```
Comment (larsoner) [inline mne/viz/_3d.py]: I don't think we want to change the logic here, but rather add `-head.fif` to this list and the one a few lines lower with `outer_skin.surf`s

Comment (larsoner) [reply mne/viz/_3d.py]: ... actually `-head.fif` should probably *only* be added to the list below, since `-head-dense.fif` is already in this list

Comment (cjayb) [reply mne/viz/_3d.py]: yup, makes sense

**Comment by agramfort** [comment]
@larsoner merge if you're happy

thx @cjayb

**Comment by larsoner** [comment]
@cjayb can you add a line to `whats_new.rst`? Then should be good to go

**Comment by agramfort** [comment]
thx @cjayb

**Pull request merged**